lib/apk.sh

```diff
@@ -20,7 +21,7 @@ _apk_init() {
 
 # apk_Q may _not_implemented
 apk_Q() {
-  case "$_TOPT" in
```

Comment:
Please keep double quotes, in this context, use double quotes or not are the same, but not using double quote can make the reader mis-understand.

I invite you to read https://unix.stackexchange.com/q/171346/38906 again 😄

Reply:
Oh, sure, that's definitely a mistake when I tried to copy & replace. I had no reason to fix that line ;)